apps/emqx_bridge/src/emqx_bridge_api.erl

```diff
@@ -693,12 +696,12 @@ get_metrics_from_local_node(BridgeType, BridgeName) ->
     ).
 
 is_enabled_bridge(BridgeType, BridgeName) ->
-    try emqx:get_config([bridges, BridgeType, binary_to_existing_atom(BridgeName)]) of
-        ConfMap ->
-            maps:get(enable, ConfMap, false)
+    try emqx_bridge:lookup(BridgeType, binary_to_existing_atom(BridgeName)) of
+        {ok, #{raw_config := ConfMap}} ->
+            maps:get(<<"enable">>, ConfMap, false);
```

Comment:
Q: do we need to consult the raw config for v2?  `enable` might not be explicitly defined, so we should take it from the transformed config, not raw, so that the default value `true` is correctly populated.

Reply:
Fixed: https://github.com/emqx/emqx/pull/11581/commits/babf5f973ae09e65d03afc148b704b1d643c10c6